test/tensor_reorder.cpp

```diff
@@ -371,6 +371,7 @@ struct reorder_invoke_param : public miopen::InvokeParams
 template <typename T>
 struct tensor_reorder_driver : tensor_reorder_base_driver
 {
+    // NOLINTBEGIN(clang-analyzer-cplusplus.NewDeleteLeaks)
```

Comment:
@atamazov clang-tidy insists there is a memory leak on Line 452. Could you help to take a look?